database/mssql/scripts/versions/v_32_ddl.sql

```diff
@@ -0,0 +1,213 @@
+SET ANSI_NULLS ON
+GO
+
+SET QUOTED_IDENTIFIER ON
+GO
+
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+
+BEGIN TRANSACTION
+GO
+
+IF @@ERROR <> 0
+   SET NOEXEC ON
+GO
+CREATE SEQUENCE permit.ORBC_LOA_NUMBER_SEQ
+    START WITH 100000
+    INCREMENT BY 1 ;
+GO
+
+CREATE TABLE [permit].[ORBC_LOA_DETAILS] (
+   LOA_ID [bigint] IDENTITY(1,1) NOT NULL,
+   LOA_NUMBER [bigint] NOT NULL  DEFAULT (NEXT VALUE FOR permit.ORBC_LOA_NUMBER_SEQ),
+   [COMPANY_ID] [int] NULL,
+   START_DATE [varchar](10) NOT NULL,
+   EXPIRY_DATE [varchar](10),
+   DOCUMENT_ID [bigint],
+   COMMENTS [nvarchar](4000),
+   [IS_ACTIVE] [bit] NOT NULL DEFAULT(1),
```

Comment:
Please change to Y or N to be consistent with other IS_ACTIVE/IS_SUSPEND flags is in the system